docs/content/latest/admin/yb-ts-cli.md

```diff
@@ -0,0 +1,208 @@
+---
+title: yb-ts-cli
+linkTitle: yb-ts-cli
+description: yb-ts-cli
+menu:
+  latest:
+    identifier: yb-ts-tool
+    parent: admin
+    weight: 2466
+isTocNested: 3
+showAsideToc: true
+---
+
+`yb-ts-cli` is a command line tool that can be used to perform an operation on tablet server (`yb-tserver`). Some of the commands perform operations similar to [`yb-admin` commands](../../../reference/configuration/yb-tserver).
+
+`yb-ts-cli` is a binary file installed with YugabyteDB and is located in the `bin` directory of the YugabyteDB home directory.
+
+## Syntax
+
+```sh
+yb-ts-cli [ --server_address=<addr> ] <command> <options>
+```
+
+- *addr*: The IP address of the tablet server. Default is `localhost`.
+- *command*: The operation to be performed. See [Commands](#commands) below.
+- *options*: The options to be applied to the command. See [Options](#options).
+
+## Command line help
+
+To display the available online help, run `yb-ts-cli` without any commands or options (flags) at the YugabyteDB home directory.
+
+```sh
+$ ./bin/yb-ts-cli
+```
+
+## Commands
+
+##### are_tablets_running
+
+If all tablets are running, returns "All tablets are running".
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<addr> ] are_tablets_running
+```
+
+##### count_intents
+
+Print the count of uncompleted intents (or [provisional records](../../../architecture/transactions/ditributed-txns/#provisional-records)). Useful for debugging and troubleshooting.
```

Comment:
I think uncommitted is more appropriate than uncompleted

also maybe end with `Useful for debugging transactional workloads.`